MODULE.bazel

```diff
@@ -1,15 +1,13 @@
 """Bazel Gazelle extension for rules_cc"""
 
-module(
-    name = "gazelle_cc",
-)
+module(name = "gazelle_cc")
 
-bazel_dep(name = "gazelle", version = "0.45.0")
+bazel_dep(name = "gazelle", version = "0.46.0")
 
 # The following deps are using the same versions as upstream gazelle
-bazel_dep(name = "rules_go", version = "0.53.0")
-bazel_dep(name = "rules_proto", version = "4.0.0")
-bazel_dep(name = "bazel_skylib", version = "1.5.0")
+bazel_dep(name = "rules_go", version = "0.58.3")
+bazel_dep(name = "rules_proto", version = "7.1.0")
+bazel_dep(name = "bazel_skylib", version = "1.8.1")
```

Comment:
Should we upgrade these as well? Gazelle 0.46.0 is still using the versions from before the diff. If we stick to them then there's a less change for incompatibilities - I mostly worry about upgrading rules_proto to 3 major versions higher then previously. 

My reasoning is that if somebody uses different (potentially incompatible) version in a project he'd need to think only about 2 versions of rules_proto to resolve - gazelle and it's own. If we use our own version then theres a third version added to the mix

If we go with the latest available then, let's remove the comment above